tests/repoFindByDate.py

```diff
@@ -158,7 +158,7 @@ def map_cfg(self, dataId, write):
         lookups = self.storage.lookup(lookupProperties='date', reference=None,
                                      dataId=dataId, template=template)
         lookups.sort()
-        if len(lookups) is not 0:
+        if len(lookups):
```

Comment:
It should be `> 0` then. `is not` is not really the operator you are looking for. An alternative could be `if lookups` which will be true if the list contains elements. I realize that `0` is a singleton in Python but that's not generally how tests against numbers are done.